### ISSUE TITLE
Please explain PluginRegistrationTool

### ISSUE THREAD

**Issue by nstratton-srx** [MicrosoftDocs/powerapps-docs#4721]
You specify using the plugin registration tool, but if like me, you haven't used it before this requires several steps to use for the first time. Please include a link with instructions on how to download and build the tool.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ad2eb047-525a-63b7-2d6e-1536a374d688
* Version Independent ID: 2b8e3e9d-c957-d43e-ceeb-c75da0857d2d
* Content: [Tutorial: Write and register a plug-in (Microsoft Dataverse) - Power Apps](https://learn.microsoft.com/en-us/power-apps/developer/data-platform/tutorial-write-plug-in)
* Content Source: [powerapps-docs/developer/data-platform/tutorial-write-plug-in.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/developer/data-platform/tutorial-write-plug-in.md)
* Service: **powerapps**
* Sub-service: **dataverse-developer**
* GitHub Login: @divkamath
* Microsoft Alias: **jdaly**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @nstratton-srx. This issue has been assigned to the doc author for review.

**Comment by phecke** [comment]
@nstratton-srx 

See [Dataverse development tools](https://learn.microsoft.com/power-apps/developer/data-platform/download-tools-nuget) for instructions to download the PRT tool using the command line interface (CLI). No building required.

**Comment by nstratton-srx** [comment]
Now I see the download in the page above. When I found that page, I took the first link in it for PRT, which looped back to your article, therefore I didn't get to the download instructions in the page. So I searched again and found the build for PRT.  

It would be helpful to have this update in your article...

To register a plug-in, you will need the Plug-in Registration tool. Instructions to download the tool can be found [here](https://learn.microsoft.com/power-apps/developer/data-platform/download-tools-nuget).
